static_src/stores/activity_store.js

```diff
@@ -66,35 +66,43 @@ class ActivityStore extends BaseStore {
     let activity;
     switch (action.type) {
       case activityActionTypes.EVENTS_FETCH:
+        this.fetching = true;
+        this.fetched = false;
         cfApi.fetchSpaceEvents(action.spaceGuid);
+        this.emitChange();
         break;
 
       case activityActionTypes.EVENTS_RECEIVED:
+        this.fetching = false;
+        this.fetched = true;
```

Comment:
Do you think there will be a potential state where the activity log will be wrong if `.fetched` gets set to true here? The activity log currently checks to see if `.fetched` is true AND the length of the data array is 0. But the events fetch request is initiated before the logs fetch request, so there can be a time when the "No recent activity" condition can be met in the component, but one of the requests is still in flight/hasn't been handled.

I think this is probably something we can live with for a while and see if there are problems with the usability before doing more work, or a lightweight check could be to have `_` variables on the store for each request and the `.fetched` can be a derived property from both?